modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/bean/InstallApplication.java

```diff
@@ -18,15 +18,16 @@
 
 package org.wso2.carbon.mdm.services.android.bean;
 
+import java.io.Serializable;
 import java.util.List;
 
 /**
  * Bean class of InstallApplication operation.
  */
-public class InstallApplicationBean {
+public class InstallApplication extends Operation implements Serializable {
```

Comment:
This too has to be re-named to something like "ApplicationInfo". Install/Uninstall are operations that can take an instance of "ApplicationInfo" in, and process it to produce desired outputs.